examples/meanio/README.md

```diff
@@ -0,0 +1,259 @@
+## MEAN.io example
+
+This example shows how to build a simple, multi-tier web application using Kubernetes and Docker.
+It is based on the [guestbook](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/guestbook) example.
+
+The example consists of:
+- A nodejs web app using MEAN framework.
+- A MongoDB server.
+
+### Step Zero: Prerequisites
+
+This example requires a kubernetes cluster. use the prerequisites of the [guestbook](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/guestbook#step-zero-prerequisites) example.
+
+### Step One: Fire up the MongoDB Server
+
+Use (or just create) the file `examples/meanio/meanio-mongodb-controller.json` which describes a single pod running a MongoDB Server:
+Note that, although the MongoDB server runs just with a single replica, we use replication controller to enforce that exactly one pod keeps running (e.g. in a event of node going down, the replication controller will ensure that the MognoDB server gets restarted on a healthy node).   This could result in data loss.
+We are using mongodb 2.6.8 for MEAN.io compatibilily.
+
+
+```js
+{
+  "id": "meanio-mongodb-controller",
+  "kind": "ReplicationController",
+  "apiVersion": "v1beta1",
```

Comment:
v1beta1 is deprecated and we are in the process of removing it.  We've moved all of the other examples to v1beta3. However, we'll be finalizing the v1 API over the next couple weeks, and then will be updating all examples again. I suggest waiting until we finalize v1, and then update this PR.